### DISCUSSION TITLE
Broadcom BCM43228 works on SpiralLinux_Plasma_11.220925_x86-64

### DISCUSSION THREAD

**Discussion by fredvej** [SpiralLinux/SpiralLinux-project#138] [General]
I booted live environment of SpiralLinux_Plasma_11.220925_x86-64.iso on my HP EliteBook 820 G1, and the wireless interface Broadcom BCM43228 worked as expected.

On Distrowatch you mentioned that you did not have any Broadcom hardware to test on, so I decided to test on my hardware and share the results.

**Comment by geckolinux** [comment]
Nice! Thanks very much, I really appreciate the report.
